dotnet-471-dev-pack/test/test.pester.ps1

```diff
@@ -0,0 +1,33 @@
+$__dir=(Get-Item $PSScriptRoot)
+$PLAN_ROOT=(Get-Item $__dir ).parent.FullName
+$CORE_ROOT=(Get-Item $PLAN_ROOT ).parent.FullName
+
+. $CORE_ROOT/results/last_build.ps1
```

Comment:
I don't think the tests need this to be sourced.
The [pending test design](https://github.com/habitat-sh/core-plans/issues/2473) (though in fairness it is still in review) would require the tests to not depend on `last_build.ps1` at all; so that we can re-run the tests against an artifacted package.

Reply:
Oh it is to get `pkg_ident`
It would be better to pass that through to Pester as a parameter

Reply:
test.ps1 and test.pester.ps1 have been refactored to receive parameters.  They no longer assume that last_build.ps1 exists.